packages/ember-data/tests/unit/model-test.js

```diff
@@ -151,6 +152,7 @@ test("it should use `_reference` and not `reference` to store its reference", fu
     });
   });
 });
+*/
```

Comment:
I think we should _ whatever we end up calling references, so probably want to keep this